tests/generic/assert_evm_address.sql

```diff
@@ -0,0 +1,14 @@
+{% test assert_evm_address(model, column_name) %}
+
+    with unit_test as
+    (
+        select
+            rlike({{ column_name }}, '0[x][0-9a-f]{40}') as assert_evm_address
```

Comment:
Can we include `column_name` as a separate column in the select here? 
That will make it easier to track down any failures.